ui/app/components/task-sub-row.js

```diff
@@ -0,0 +1,76 @@
+import Ember from 'ember';
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { computed } from '@ember/object';
+import { alias } from '@ember/object/computed';
+import { task, timeout } from 'ember-concurrency';
+import { tracked } from '@glimmer/tracking';
+
+export default class TaskSubRowComponent extends Component {
+  @service store;
+  @service router;
+  @service('stats-trackers-registry') statsTrackersRegistry;
+
+  constructor() {
+    super(...arguments);
+    // Kick off stats polling
+    const allocation = this.task.allocation;
+    if (allocation) {
+      this.fetchStats.perform();
+    } else {
+      this.fetchStats.cancelAll();
+    }
+  }
```

Comment:
`question` will this stop polling when the component unmounts?

Reply:
Yep; inherited from our other stats fetchers, like task-row. On unmount, or even on hide, this will stop its stats polling.